docs/version0.27.md

```diff
@@ -68,4 +68,8 @@ The [_JAVA_OPTIONS environment variable](cmdline_specifying.md) (with different
 
 Global lock reservation is now enabled by default. This is an optimization targeted towards more efficient handling of locking and unlocking Java&trade; objects. The older locking behavior can be restored via the `-XX:-GlobalLockReservation` option. See [-XX:[+|-]GlobalLockReservation](xxgloballockreservation.md) for more details.
 
+### Increase default OS stack size on PPC64 platforms
+
+The default OS stack size on AIX and Linux PPC64 is increased from 256KB to 512KB. To manually set the OS stack size one can use the -Xmso command line option.
+
```

Comment:
Can we please spell out "OS" again here, also change the second sentence to remove "one can" as that is a construct that non-native English speakers might have problems with, and add a link. I suggest `You can change the operating system stack size by using the [-Xmso](xmso.md) option.`

Can you please also add a link to the new section in the list at the top of the version0.27.md file - MkDocs will automatically generate an id for the section from the title, so the links needs to be the title in lowercase with dashes instead of spaces. Something like:

`- [Increase in default operating system stack size on PPC64 platforms](#increase-in-default-operating-system-stack-size-on-ppc64-platforms)`